package.json

```diff
@@ -4,7 +4,8 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "cross-env NODE_ENV=development mocha src/__tests__/* --require @babel/register --require @babel/polyfill",
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
```

Comment:
@ola357 I get this error when I run coverage 👇🏽
<img width="1440" alt="Screenshot 2019-08-20 at 11 06 04 AM" src="https://user-images.githubusercontent.com/27569442/63338372-95304980-c33a-11e9-90db-6e6c227e34f5.png">
.

Reply:
Error has been fixed

Reply:
the error has been fixed, PR is ready for merging @rovilay